amplify/cli.json

```diff
@@ -24,7 +24,7 @@
       "useenabledmfas": false
     },
     "codegen": {
-      "useappsyncmodelgenplugin": false,
+      "useappsyncmodelgenplugin": true,
```

Comment:
Is it automatically changed or manually changed?

Reply:
Again, automatically generated.